randovania/games/prime_hunters/generator/bootstrap.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from randovania.game_description.db.configurable_node import ConfigurableNode
+from randovania.game_description.requirements.resource_requirement import ResourceRequirement
+from randovania.games.prime_hunters.layout import HuntersConfiguration
+from randovania.games.prime_hunters.layout.force_field_configuration import LayoutForceFieldRequirement
+from randovania.resolver.bootstrap import Bootstrap
+from randovania.resolver.energy_tank_damage_state import EnergyTankDamageState
+
+if TYPE_CHECKING:
+    from random import Random
+
+    from randovania.game_description.db.pickup_node import PickupNode
+    from randovania.game_description.game_description import GameDescription
+    from randovania.game_description.game_patches import GamePatches
+    from randovania.generator.pickup_pool import PoolResults
+    from randovania.resolver.damage_state import DamageState
+
+
+def is_boss_location(node: PickupNode, config: HuntersConfiguration) -> bool:
+    octolith = node.extra.get("entity_type_data", {}).get("model_id")
+    if octolith is not None and octolith == 8 and config.octoliths:
+        return True
+
+    return False
+
+
+class HuntersBootstrap(Bootstrap[HuntersConfiguration]):
+    def create_damage_state(self, game: GameDescription, configuration: HuntersConfiguration) -> DamageState:
+        return EnergyTankDamageState(
+            100,
```

Comment:
You don't start with 99 health in Hunters? :O

Reply:
No I just never changed this